docs/guides/admin/docs/releasenotes/filter-admin-ui-users.txt

```diff
@@ -1 +0,0 @@
-- [#2019](https://github.com/opencast/opencast/pull/2019): The full list of Opencast users can now be reduced in the admin UI's filtering tool using a regular expression
```

Comment:
Why does this delete any release notes?

Reply:
I was already wondering whether that was the right way to go - do we want to keep these for older releases? If I remember correctly, the extra files are just for the release managers so they can easily create the release notes, so having them in there for multiple releases without distinction might be very confusing? On the other hand, these are extra files so merging forward is easier, if I remember correctly, so... Idk, I'm confused. These wouldn't be new features in OC11, so do we want to keep them in here or not? I'm fine with either.